modules/gles/Makefile.am

```diff
@@ -12,6 +12,7 @@ XCAM_GLES_LIBS =                     \
     $(NULL)
 
 xcam_gles_sources =                  \
+    gl_error.cpp                     \
```

Comment:
no need to separate new file gl_error.h / .cpp.
move all gl related utilities to gles_std.h or gles_common.h